README.md

```diff
@@ -22,13 +22,13 @@ uses wget to pull all the data from those url text files and deposit them in the
 
 Requirements
 ------------
-IATI Registry Refresher requires PHP version 5.2.0 or later.
+IATI Registry Refresher requires python 3.
 
 It also requires curl.
 on Ubuntu 
 ```
 sudo apt-get install curl
-sudo apt-get install php5-curl php5-cli
+sudo apt-get install python-pip python-dev
 ```
 
```

Comment:
It would be great if we had some documentation regarding a virtual environment here. Whilst we aren't installing any dependencies, simply using a python3 environment to run `git.sh` makes the tool a little more os agnostic and predictable for different users (given that some operating systems use Python 2.7 by default, still).

```
python3 -m venv pyenv
```

Reply:
Happy to add it.

Personally I use [pipenv](https://pipenv.readthedocs.io) more and more, so I feel a bit averse to being prescriptive about this sort of thing. But I guess it’s fine as a default setup.

Reply:
Fixed in 03bf889.

Reply:
Thanks! I see your point. In the interest of not being prescriptive, but also being instructive enough to those that may not be familiar with virtual environments, some text alluding to  '... or your preferred virtual environment' could cover both.